src/@sdk/helpers/LocalStorageHandler/LocalStorageHandler.ts

```diff
@@ -0,0 +1,65 @@
+import { NamedObservable } from "../NamedObservable";
+import {
+  ICheckoutModel,
+  ILocalStorageHandler,
+  IPaymentModel,
+  LocalStorageItems,
+} from "./types";
+
+class LocalStorageHandlerProxy extends NamedObservable<LocalStorageItems> {
```

Comment:
I would like to see docstring about 'how to remove cached entry'. It was not clear to me without reading the code.

Reply:
Also it's unclear for me when should I use get/set Item, and when get/set Object and why there is distinction between those.

Reply:
I have added docstring